### PR TITLE
refactor[script]: Updated infra-setup stage in konvoy-onprem 

### PR DESCRIPTION
**Pull request by w3aman**
Signed-off-by: Aman Gupta <aman.gupta@mayadata.io>

1. Added jobs in infra-setup stage in konvoy-onprem.
2. Deleted unwanted/unused files.
3. Update the gitlab-ci.yaml according to the above changes.

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ stages:
   # - APP-DEPROVISION    
   - CLUSTER-CLEANUP
 
-konvoy-cluster:
+P7Q4-konvoy-cluster:
   image: openebs/openshift-ci:latest
```
Comment (gprasath) [inline .gitlab-ci.yml]: update image

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -9,7 +9,7 @@ stages:
   # - APP-DEPROVISION    
   - CLUSTER-CLEANUP
 
-konvoy-cluster:
+P7Q4-konvoy-cluster:
```
[the remark this answers is not in the document]
Comment (w3aman) [reply .gitlab-ci.yml]: Done!!

File: openebs-konvoy-e2e/pipelines/OpenEBS-base/stages/2-Infra-setup/OQEW-create-nfs-provisioner/nfsv4-provision [hunk 3]
```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+set -x
+pod() {
+
+sshpass -p $pass ssh -o StrictHostKeyChecking=no $user@$ip -p $port ssh root@10.23.1.10 'oc adm policy add-scc-to-user privileged system:serviceaccount:app-nfsv4-ns:nfs-provisioner'
```
Comment (shashank855) [inline openebs-konvoy-e2e/pipelines/OpenEBS-base/stages/2-Infra-setup/OQEW-create-nfs-provisioner/nfsv4-provision]: Remove oc adm commands.

File: openebs-konvoy-e2e/pipelines/OpenEBS-base/stages/2-Infra-setup/OQEW-create-nfs-provisioner/nfsv4-provision [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+sshpass -p $pass ssh -o StrictHostKeyChecking=no $user@$ip -p $port ssh root@10.23.1.10 'oc adm policy add-scc-to-user anyuid system:serviceaccount:app-nfsv4-ns:nfs-provisioner'
```
Comment (shashank855) [inline openebs-konvoy-e2e/pipelines/OpenEBS-base/stages/2-Infra-setup/OQEW-create-nfs-provisioner/nfsv4-provision]: remove oc adm command

File: openebs-konvoy-e2e/pipelines/OpenEBS-base/stages/2-Infra-setup/XJGT-openebs-deploy/infra-setup [hunk 5]
```diff
@@ -2,14 +2,14 @@
 set -x
 pod() {
 sshpass -p $pass ssh -o StrictHostKeyChecking=no $user@$ip -p $port 'git clone -b konvoy-onprem https://github.com/mayadata-io/e2e-konvoy.git'
```
Comment (shashank855) [inline openebs-konvoy-e2e/pipelines/OpenEBS-base/stages/2-Infra-setup/XJGT-openebs-deploy/infra-setup]: remove git clone from here as the repo is already been cloned in the cluster-setup stage

File: openebs-konvoy-e2e/pipelines/OpenEBS-base/stages/2-Infra-setup/OQEW-create-nfs-provisioner/nfsv4-provision (hunk 3 above)
Comment (w3aman) [reply openebs-konvoy-e2e/pipelines/OpenEBS-base/stages/2-Infra-setup/OQEW-create-nfs-provisioner/nfsv4-provision]: Removed..

File: openebs-konvoy-e2e/pipelines/OpenEBS-base/stages/2-Infra-setup/OQEW-create-nfs-provisioner/nfsv4-provision (hunk 4 above)
Comment (w3aman) [reply openebs-konvoy-e2e/pipelines/OpenEBS-base/stages/2-Infra-setup/OQEW-create-nfs-provisioner/nfsv4-provision]: Done!

File: openebs-konvoy-e2e/pipelines/OpenEBS-base/stages/2-Infra-setup/XJGT-openebs-deploy/infra-setup (hunk 5 above)
Comment (w3aman) [reply openebs-konvoy-e2e/pipelines/OpenEBS-base/stages/2-Infra-setup/XJGT-openebs-deploy/infra-setup]: Ok..

**Pull request merged**